controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -575,6 +593,17 @@ func validateControlPlaneSpec(rosaClient *rosa.RosaClient, rosaScope *scope.ROSA
 		return &message, nil
 	}
 
+	// sanitize Tags
+	tagsList := map[string]string{}
+
+	if len(rosaScope.ControlPlane.Spec.AdditionalTags) > 0 {
+		for tagKey, tagVal := range rosaScope.ControlPlane.Spec.AdditionalTags {
+			tagsList[strings.TrimSpace(tagKey)] = strings.TrimSpace(tagVal)
+		}
+	}
+
+	rosaScope.ControlPlane.Spec.AdditionalTags = tagsList
```

Comment:
this is not needed with the validation in the webhook.

Reply:
removed